src/global/Constants.h

```diff
@@ -71,3 +73,6 @@ static constexpr uint8_t NUM_COMPRESSION_PREFIXES = 127;
 // compression has been applied to  a word
 static const uint8_t NO_PREFIX_CHAR =
     MIN_COMPRESSION_PREFIX + NUM_COMPRESSION_PREFIXES;
+
+static constexpr auto DEFAULT_QUERY_COMPUTATIION_TIMEOUT = 30s;
+static constexpr auto DEFAULT_STRING_TRANSLATION_TIMEOUT = 30s;
```

Comment:
I think I would set both to 1 minute and I think it would make sense to be able to override them with a query parameter. It's not too rare that we use QLever to extract things like "all names of all entities" which is necessarily slow but works quite well at the moment.